rules/windows/process_creation/proc_creation_win_pressanykey_lolbin_execution.yml

```
@@ -0,0 +1,27 @@
+title: Visual Studio NodejsTools PressAnyKey Arbitrary Binary Execution
+id: a20391f8-76fb-437b-abc0-dba2df1952c6
+related:
+    - id: 65c3ca2c-525f-4ced-968e-246a713d164f
+      type: similar
+status: test
+description: Detects a certain command line flag combination used by Microsoft.NodejsTools.PressAnyKey.exe that can be used to execute any other binary
```

The rule doesn't really use the command line flags:

```suggestion
description: Detects child processes of Microsoft.NodejsTools.PressAnyKey.exe that can be used to execute any other binary
```